src/gui/addnewtorrentdialog.cpp

```diff
@@ -117,7 +117,7 @@ AddNewTorrentDialog::AddNewTorrentDialog(QWidget *parent)
     // Signal / slots
     connect(ui->adv_button, SIGNAL(clicked(bool)), SLOT(showAdvancedSettings(bool)));
     connect(ui->doNotDeleteTorrentCheckBox, SIGNAL(clicked(bool)), SLOT(doNotDeleteTorrentClicked(bool)));
-    editHotkey = new QShortcut(QKeySequence("F2"), ui->contentTreeView, 0, 0, Qt::WidgetShortcut);
+    editHotkey = new QShortcut(QKeySequence(Qt::Key_F2), ui->contentTreeView, 0, 0, Qt::WidgetShortcut);
```

Comment:
Should i put a parenthesis around combination of keys?
`QShortcut *switchSearchShortcut = new QShortcut((Qt::ALT + Qt::Key_2), this);`
or
`QShortcut *switchSearchShortcut = new QShortcut(Qt::ALT + Qt::Key_2, this);`